index.bs

```diff
@@ -107,19 +113,67 @@ The {{GravitySensor}} class is an {{Accelerometer}}'s subclass. The {{GravitySen
 
 The <dfn>gravity</dfn> is a force that attracts an object to the center of the earth, or towards any other physical object having mass.
 
-Note: The <dfn>local coordinate system</dfn> of a mobile device is usually defined relative to
-the device's screen when the device in its default orientation (see figure below).
+Coordinate systems {#coordinate-systems}
+------------------
 
-<img src="images/accelerometer_coordinate_system.png" srcset="images/accelerometer_coordinate_system.svg" style="display: block;margin: auto;" alt="Accelerometer coordinate system.">
+The [=local coordinate system=] for the spatial sensor classes in this specification
+can be defined as either [=device coordinate system=] or [=screen coordinate system=].
+
+The <dfn export>device coordinate system</dfn> is defined as a three dimensional
+Cartesian coordinate system (x, y, z) which is bound to the physical device.
+For devices with display the origin of the [=device coordinate system=] is in the center
+of the device display. If the device is held in a default position the Y-axis points
```

Comment:
"a default position" sounds wake... will it have multiple? in its default position